src/Async/Projections/ProjectionsManager.php

```diff
@@ -160,7 +160,7 @@ public function getQueryAsync(string $name, ?UserCredentials $userCredentials =
     public function updateQueryAsync(
         string $name,
         string $query,
-        bool $emitEnabled = false,
+        bool $emitEnabled = null,
```

Comment:
```suggestion
        ?bool $emitEnabled = null,
```

Null means no change I'm guessing?

Reply:
yes